main.html

```diff
@@ -48,35 +48,194 @@ <h4 class="tour-logo_bot">2023</h4>
+                soul artists and emerging talents. 
+              </p>
+  
+              <h3 class="main_section_date">2023.10.29(THU) - 31(SAT)</h3>
+  
+              <h4 class="main_section_place">
+                @COFRE DE PEROTE forum - Perote, México
+              </h4>
+            </div>
+          </section>
+        </div>
+        </section>
+      </div>
+      <section class="main_program" id="program">
+        <h2 class="section_title">Festival Program</h2>
+        <div class="div-line"></div>
+
+        <div class="main_program_container">
+          <article class="main_program_container_card">
+            <img
+              src="./assets/logos/forums.PNG"
+              alt="Concerts Logo"
+              class="main_program_container_card_img"
+            />
+            <h3>Concerts</h3>
+
+            <p>
+              Enjoy the best R&B Music concerts with the most prominent R&B artists from 2023
+            </p>
+          </article>
+
+          <article class="main_program_container_card">
+            <img
+              src="./assets/logos/open-mic.PNG"
+              alt="Open Mic Logo"
+              class="main_program_container_card_img"
+            />
+            <h3>Open Mic</h3>
+            <p>
+              We want to listen what you got! Come and show us your talent an experience the R&B Music World Tour
+            </p>
+          </article>
+
+          <article class="main_program_container_card">
+            <img
+              src="./assets/logos/networking.PNG"
+              alt="Forums Logo"
+              class="main_program_container_card_img"
+            />
+            <h3>Forums</h3>
+            <p>
+              Our special Guests will be sharing knowledge and experiences with the audience
+            </p>
+          </article>
+
+          <article class="main_program_container_card">
+            <img
+              src="./assets/logos/family.PNG"
+              alt="Family Logo"
+              class="main_program_container_card_img"
+            />
+            <h3>Family</h3>
+            <p>
+              This experience is for everyone!
+            </p>
+          </article>
+
+          <article class="main_program_container_card">
+            <img
+              src="./assets/logos/food.PNG"
+              alt="Food Logo"
+              class="main_program_container_card_img"
+            />
+            <h3>Food</h3>
+            <p>
+              We invited the best chefs and food trucks from Mexico to make your experience even better
+            </p>
+          </article>
+        </div>
+
+        <a class="main_program_ext-link section_ext-link" href="#">
+          <h3> Discover the Experience</h3>
+        </a>
+
+        <button class="main_program_button">
+          Be part of the R&B Music World Tour 2023
+        </button>
+      </section>
+
+      <!------LINEUP - DYNAMIC ------->
+      
+       <section class="lineup_section" id="lineup"></section>
```

Comment:
- [ ]  Hi, You have done a wonderful job. **Can you go back to the suggestion made by the previous reviewer, and implement it? He suggested that you should implement the button `MORE` and make it match the original design**. We request this because the page should almost look identical to the original design(Ref. Project requirements).
- Original design:
![Capture](https://github.com/rudicarrilloypr/capstone-module-1/assets/63972670/b56e8181-b22d-4cc6-8513-55ffdacf8dd7)
- Your Design:
![Capture](https://github.com/rudicarrilloypr/capstone-module-1/assets/63972670/ea364219-32a4-44f3-9db1-6d8933dda51d)

Reply:
the button its made as the first revieer told me....

![IMG_5707](https://github.com/rudicarrilloypr/capstone-module-1/assets/125106965/b37d08b3-44c0-4ac5-a85c-e009c029d7f0)